File: IAIso-v5.0/core/iaiso-php/src/Policy/SumAggregator.php
<?php

declare(strict_types=1);

namespace IAIso\Policy;

/** Sum of per-execution pressures. */
final class SumAggregator implements Aggregator
{
    public function name(): string { return 'sum'; }

    public function aggregate(array $pressures): float
    {
        return array_sum($pressures);
    }
}

File: IAIso-v5.0/core/iaiso-java/iaiso-policy/src/main/java/io/iaiso/policy/SumAggregator.java
package io.iaiso.policy;

import java.util.Collections;
import java.util.Map;

/** Sum of per-execution pressures. */
public final class SumAggregator implements Aggregator {
    @Override public String name() { return "sum"; }
    @Override public double aggregate(Map<String, Double> p) {
        double total = 0.0;
        for (Double v : p.values()) total += v;
        return total;
    }
}
